src/components/Reactions/ReactionTooltipContent.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import styles from '../../styles/styles';
+import {withPersonalDetails} from '../OnyxProvider';
+import * as PersonalDetailsUtils from '../../libs/PersonalDetailsUtils';
+import Text from '../Text';
+import withCurrentUserPersonalDetails, {
+    withCurrentUserPersonalDetailsPropTypes,
+} from '../withCurrentUserPersonalDetails';
+import compose from '../../libs/compose';
+import withLocalize from '../withLocalize';
+
+const propTypes = {
+    /**
+     * A list of emoji codes to display in the tooltip.
+     */
+    emojiCodes: PropTypes.arrayOf(PropTypes.string).isRequired,
+
+    /**
+     * The name of the emoji to display in the tooltip.
+     */
+    emojiName: PropTypes.string.isRequired,
+
+    /**
+     * A list of account IDs to display in the tooltip.
+     */
+    accountIDs: PropTypes.arrayOf(PropTypes.string).isRequired,
+
+    ...withCurrentUserPersonalDetailsPropTypes,
+};
+
+const ReactionTooltipContent = (props) => {
+    const users = PersonalDetailsUtils.getPersonalDetailsByIDs(props.accountIDs, true);
+    const namesString = _.filter(_.map(users, user => user && user.displayName), n => n).join(', ');
```

Comment:
What is the purpose of the filter here? It doesn't look like it's doing anything?

<img width="574" alt="2023-03-14_12-56-10" src="https://user-images.githubusercontent.com/42391420/225080265-aee60030-084b-44e8-857d-b0ca982db278.png">

Reply:
`user && user.displayName` can produce undefined, if user is undefined. I don't recall exactly how but there were cases where the user wasn't in the personal details yet and that was causing issues